automation/project/Dependencies.scala

```diff
@@ -7,11 +7,11 @@ object Dependencies {
   val akkaHttpV     = "10.2.0"
   val jacksonV      = "2.12.3"
 
-  val serviceTestV = "0.20-d999f210-SNAP"
-  val workbenchGoogleV = "0.21-5c9c4f6"
-  val workbenchGoogle2V = "0.21-e17afdf"
-  val workbenchModelV  = "0.14-65bba14"
-  val workbenchMetricsV  = "0.5-64a7b29"
+  val serviceTestV = "0.20-21408a9"
+  val workbenchGoogleV = "0.21-21408a9"
+  val workbenchGoogle2V = "0.22-21408a9"
+  val workbenchModelV  = "0.14-21408a9"
+  val workbenchMetricsV  = "0.5-21408a9"
```

Comment:
per Qi's suggestion, I updated all of these to the same version which makes managing dependencies between these libraries easier

Reply:
There are also a number of changes that were required when updating the `serviceTest` version because it now uses cats-effect 3. I'll call out those changes in the PR